lib/services/northBound/contextServer.js

```diff
@@ -482,6 +704,132 @@ function handleQuery(req, res, next) {
     ], handleQueryContextRequests);
 }
 
+/**
+ * Express middleware to manage incoming QueryContext requests using NGSIv2.
```

Comment:
"QueryContext" is a NGSIv1 term. Let's say just "query context requests using NGSIv2" ;)

Reply:
Similar one for "UpdateContext" (if any).

Reply:
Fixed with fe842129a6cf92571b6908dbb42d5645327b1f34